console/application.go

```diff
@@ -229,3 +236,258 @@ func flagsToCliFlags(flags []command.Flag) []cli.Flag {
 
 	return cliFlags
 }
+
+func argumentsToCliArgs(args []command.Argument) ([]cli.Argument, error) {
+	len := len(args)
+	if len == 0 {
+		return nil, nil
+	}
+	cliArgs := make([]cli.Argument, 0, len)
+	previousIsRequired := true
+	for _, v := range args {
+		if v.MinOccurrences() != 0 && !previousIsRequired {
+			return nil, errors.ConsoleCommandRequiredArgumentWrongOrder.Args(v.ArgumentName())
+		}
+		if v.MinOccurrences() != 0 {
+			previousIsRequired = true
+		} else {
+			previousIsRequired = false
+		}
+		switch arg := v.(type) {
+		case *command.ArgumentFloat32:
+			cliArgs = append(cliArgs, &cli.Float32Args{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+			})
+		case *command.ArgumentFloat64:
+			cliArgs = append(cliArgs, &cli.Float64Args{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+			})
+		case *command.ArgumentInt:
+			cliArgs = append(cliArgs, &cli.IntArgs{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+			})
+		case *command.ArgumentInt8:
+			cliArgs = append(cliArgs, &cli.Int8Args{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+			})
+		case *command.ArgumentInt16:
+			cliArgs = append(cliArgs, &cli.Int16Args{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+			})
+		case *command.ArgumentInt32:
+			cliArgs = append(cliArgs, &cli.Int32Args{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+			})
+		case *command.ArgumentInt64:
+			cliArgs = append(cliArgs, &cli.Int64Args{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+			})
+		case *command.ArgumentString:
+			cliArgs = append(cliArgs, &cli.StringArgs{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+			})
+		case *command.ArgumentTimestamp:
+			cliArgs = append(cliArgs, &cli.TimestampArgs{
+				Name:      arg.Name,
+				UsageText: arg.Usage,
+				Value:     arg.Value,
+				Min:       arg.MinOccurrences(),
+				Max:       arg.MaxOccurrences(),
+				Config: cli.TimestampConfig{
+					Layouts: []string{time.RFC3339},
```

Comment:
How about adding a time format configuration option to `command.ArgumentTimestamp`? This would allow users to specify their desired format, making it more flexible and user-friendly.